lib/test/t_FunctionalChaosValidation_ishigami.expout

```diff
@@ -0,0 +1,28 @@
+basisDimension = 100
```

Comment:
please avoid this file

you can compute LOO naively and using analytical formula & compare them using assert!
Same for k-fold

Reply:
This can be done in lib/test/CMakeLists.txt using the IGNOREOUT macro.

Reply:
Fixed at 2df23ef8e1fd48325d0a47642892d953c24e9590